docs/meshstack.billing.md

```diff
@@ -1,63 +1,37 @@
+meshStack collects all usage information from public cloud providers, each of which has its own metering processes. For private clouds like OpenStack, OpenShift, and Cloud Foundry, meshStack has its own metering system to for example accurately calculate how many RAM hours and vCPU hours a virtual machine consumes over a given period.
 
-Cloud resources have many different traits, i.e. we saw that a virtual machine has RAM and vCPU.
-A Product Catalog defines which of these traits are relevant for metering and how their usage is calculated. Typically
-usage is the product of a quantity and a duration, i.e. a single vCPU used for an hour. But their may be other usage units as well that consist only of quantities (i.e. bytes transferred over the network) or a duration (i.e. resource usage hour).
+Cloud resources exhibit various traits. For instance, a virtual machine typically includes both RAM and vCPU. A crucial component of meshStack is the Product Catalog, which is a list of traits that are relevant for you and prices for them. With Product Catalof you can set not only usage prices but also additional prices to cover licensing costs and maintenance of Platform. Please read more information about the Product Catalog [here](meshstack.billing-configuration.md#setting-internal-prices).
 
-A product catalog also contains pricing rules that determine the cost for a particular resource usage.
 
 ## Billing
 
 > Billing is the process of attributing resource usage to cloud tenants and creating appropriate invoices.
 
-There are two principal steps to the billing process. The first is creating periodic (e.g. monthly) [Tenant Usage Reports](meshcloud.project-metering.md) that
-aggregate cloud resource usage data for tenants. meshStack makes [Tenant Usage Reports](meshcloud.project-metering.md) available to all
-involved users, i.e. workspace & project owners, platform operators and partners.
-
-The second is invoicing the application team (represented by a meshWorkspace) according to agreed terms for this usage. This may also involve applying additional
-pricing and discount rules to aggregated usage reports, i.e. volume discounts.
+meshStack is creating monthly [Tenant Usage Reports](meshcloud.project-metering.md) that aggregate cloud resource usage data for each tenant over a specific period. These reports not only summarize resource consumption but can also include additional pricing information, as mentioned earlier, along with discounts, such as volume discounts.
 
-### Public cloud billing with meshcloud
+Furthermore, instead of providing a single, large invoice as cloud providers do, meshStack prepares invoices tailored to dedicated teams based on their usage, so that they will have a clear overview of cloud costs. This invoce called chargeaback statment, which is aslo used to correctly allocate costs to teams. 
```

Comment:
```suggestion
Furthermore, instead of providing a single, large invoice as cloud providers do, meshStack prepares invoices tailored to dedicated teams based on their usage, so that they will have a clear overview of cloud costs. This invoice is called a chargeback statement, which is also used to correctly allocate costs to teams. 
```